src/sentry/static/sentry/app/components/dropdownControl.tsx

```diff
@@ -71,12 +88,12 @@ class DropdownControl extends React.Component {
               {this.renderButton(isOpen, getActorProps)}
               <Content
                 {...getMenuProps()}
-                alignMenu={alignRight ? 'right' : 'left'}
+                alignMenu={alignMenu}
                 width={menuWidth}
-                menuOffset={menuOffset}
```

Comment:
couldn't find any references to menuOffset outside of the storybook

Reply:
It was likely removed in other refactorings :clap: